Gemfile

```diff
@@ -1,6 +1,6 @@
 source "https://rubygems.org"
 # Jekyll Theme
-gem 'jumbo-jekyll-theme', "5.5.1"
+gem 'jumbo-jekyll-theme', "5.5.4"
```

Comment:
I thought we were trying to reduce the number of different theme versions in use? This would add a new one:
[DockerFile](https://github.com/linaro-its/jekyll-build-container/blob/615a4189baf8518498ceea9a711b9ddfa81a06ea/Dockerfile) for `jekyll-build-container`.

Reply:
I've added a new feature to the theme to add a video to feature_blocks. Unless you have a better solution?

Reply:
Then you need to submit a PR against the `jekyll-build-container`. :smiley:

Reply:
NVM! PR [#19](https://github.com/linaro-its/jekyll-build-container/pull/19/files)